utils/hct/hcttest.cmd

```diff
@@ -265,7 +265,20 @@ if "%TEST_CLEAN%"=="1" (
 )
 
 if "%TEST_MANUAL_FILE_CHECK%"=="1" (
-  set TEST_USE_LIT=0
+  echo %MANUAL_FILE_CHECK_PATH%|find "\HLSLFileCheck\" >nul
```

Comment:
This is case-sensitive, but the file system on Windows is not.

Reply:
Fixed with -i.